MiniDumper/MiniDumper.cs

```diff
@@ -178,14 +200,85 @@ public void PrintDebugString(OUTPUT_DEBUG_STRING_INFO outputDbgStrInfo)
             }
         }
 
+        public void MemoryCommitThreshold(uint? commitThreshold, uint? commitDrops, int NumberOfDumps, ref bool isTimerSet)
+        {
+            isTimerSet = true;
+            CommitThreshold = commitThreshold * 1048576;
+            CommitDrops = commitDrops * 1048576;
+
+            hProcess = ProcessNativeMethod.OpenProcess(ProcessAccessFlags.QueryInformation, false, pid);
+            if (hProcess.IsInvalid)
+                throw new ArgumentException(String.Format("Unable to open process {0}, error {x:8}", pid, Marshal.GetLastWin32Error()));
+
+            OnTimedEvent();
+
+            if (Debugger.IsPollingCompleted)
+                return;
+
+            pollingTimer.Interval = 1000;
+            pollingTimer.Elapsed += OnTimedEvent;
+            pollingTimer.AutoReset = true;
+            pollingTimer.Enabled = true;
+        }
+
+        private void OnTimedEvent(Object source, ElapsedEventArgs e)
+        {
+            OnTimedEvent();
+        }
+
+        private void OnTimedEvent()
+        {
+            processCommit = GetProcessCommit(hProcess);
+            if ((CommitThreshold.HasValue && processCommit >= CommitThreshold) || (CommitDrops.HasValue && processCommit >= CommitDrops))
+            {
+                Debug.Write(CommitThreshold.HasValue ? $"Commit:\t >= {CommitDrops / 1024 / 1024}MB" : $"Commit:\t <= {CommitDrops / 1024 / 1024}MB");
+                Console.WriteLine("Commit {0}", processCommit / 1024 / 1024);
```

Comment:
Please make this message more meaningful (as in the line above)

Reply:
I just followed as procdump. please suggest some message .
![image](https://user-images.githubusercontent.com/8208927/36843445-f928fb8e-1d74-11e8-98c3-5581d8082ae1.png)

Reply:
Ach, OK. That is fine then, but please add 'Mb' at the end. Someone might not know what is the memory unit.